.github/workflows/e2e-matrix.yaml

```diff
@@ -83,6 +83,8 @@ jobs:
             region: ${{ inputs.region }}
           - name: IPv6
             region: ${{ inputs.region }}
+          - name: Termination
+            region: ${{ inputs.region }}
```

Comment:
Do we need another suite for this? This is another cluster we create, which impacts our test concurrency limits.  Is there another suite this can fit well into? Or is this because the Integration test suite is already too big?

I'm wondering if we can move some of the integration stuff to termination and save some time on the avg test suite.

Reply:
Sure, can do that.